src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -120,7 +120,6 @@ public FeedVersion() {
     /**
      * The feed source this is associated with
      */
-    @JsonView(JsonViews.DataDump.class)
```

Comment:
Why is this removed?

Reply:
That field is not included otherwise except in dumps. The UI "Download" button in https://github.com/ibi-group/datatools-ui/pull/792 expects the `feedSourceId` field.

Reply:
Ah this makes sense thank you